add zh_CN.mo zh_CN.po file

makeitsofun opened the pull request:
New Chinese translation of po and mo files has been added

Tetrakern commented:
Hello! Thanks for your contribution! I have seen it and I will merge it, I'm just currently refactoring a lot, which also includes some minor translation updates.

Tetrakern commented:
@makeitsofun Hello! I have updated a few translations for the next release. Just some minor things, if you want to add them before I merge your PR.

makeitsofun commented:
@Tetrakern Hello! based on the latest content of the fictioneer.pot file, I updated the zh_CN translation, conducted a full content check and alignment, and fixed some of the previous translation issues that involved character encoding.

Finally, it was also tested and used on the website: [https://www.wormread.com](https://www.wormread.com) You can check the results.
